bokeh/colors/color.py

```diff
@@ -22,6 +22,7 @@
 
 # Standard library imports
 from abc import ABCMeta, abstractmethod
+import colorsys
 from typing import TYPE_CHECKING, Type, TypeVar
```

Comment:
```suggestion
import colorsys
from abc import ABCMeta, abstractmethod
from typing import TYPE_CHECKING, Type, TypeVar
```

isort didn't like this in the codebase tests. It looks to sort `import xyz` statements alphabetically, then sort `from foo import bar` alphabetically

Reply:
Thanks! Applied the changes as suggested.

However, the real fails we can't figure out is the clean import tests. Like in `test_no_ipython_common.py`, it says `AssertionError: IPython imported in common modules`. However, we never specified any files or anaconda to import ipython. Same goes for selenium and tornado.

Reply:
I see [several other failed runs with the same error](https://github.com/bokeh/bokeh/actions), so I feel confident saying it's not you, it's us 😄